modules/installation-aws-hcp-sd-user-infra-rhcos-ami.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+
+[id="installation-aws-hcp-user-infra-rhcos-ami_{context}"]
+= ROSA AMIs for the AWS infrastructure
+
+Red Hat provides {op-system-first} AMIs that are valid for the various AWS regions and instance architectures that you can manually specify for your {product-title} nodes.
```

Comment:
I think something in this sentence needs to be reordered to make more sense unless it is the "instance architectures" that you manually specify for the nodes.